src/stores/connectors/eth1/delete.go

```diff
@@ -20,25 +20,23 @@ func (c Connector) Delete(ctx context.Context, addr ethcommon.Address) error {
 	}
 
 	acc, err := c.db.Get(ctx, addr.Hex())
-	if err != nil {
-		return err
-	}
-
-	err = c.db.RunInTransaction(ctx, func(dbtx database.ETH1Accounts) error {
-		err = dbtx.Delete(ctx, addr.Hex())
+	if err == nil {
```

Comment:
According to what it is described at the rest-api doc https://restfulapi.net/idempotent-rest-apis/ 
```
HTTP DELETE
When you invoke N similar DELETE requests, first request will delete the resource and response will be 200 (OK) or 204 (No Content). Other N-1 requests will return 404 (Not Found). Clearly, the response is different from first request, but there is no change of state for any resource on server side because original resource is already deleted. So, DELETE is idempotent.
```

it should fail with 404 when the asset is already deleted, therefore previous behaviour was okay.

Reply:
Yes right, I did roll back